next.clayui.com/content/docs/css/components/list.md

```diff
@@ -0,0 +1,1343 @@
+---
+title: "List"
+description: "List is a visual representation of a dataset that provides more flexibility for arranging the data than a table and is less visually explicit than a card view."
+---
+
+<div class="nav-toc">
+
+- [Variations](#variations)
+	- [Bordered](#bordered)
+    - [Default](#default)
+    - [Notification](#notification)
+- [Group Elements](#group-elements)
+    - [Header](#header)
+	- [Items](#items)
+	    - [Items as Links and Buttons](#items-as-links-and-buttons)
+	    - [Actions on Hover](#actions-on-hover)
+	        - [Anchors](#anchors)
+            - [Buttons](#buttons)
+	    - [Active](#active)
+	    - [Display Types](#display-types)
+        - [Active Display Types](#active-display-types)
+- [Examples of use](#examples-of-use)
```

Comment:
I think we do not have this session, we can move this to the beginning and create just with the name `Example`.